housing_cooperative_base/data/cron.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">  <!-- fixme -->
+        <record forcecreate="True" id="ir_cron_compute_state" model="ir.cron">
+            <field name="name">Compute premise states</field>
+            <field name="model_id" ref="model_hc_premise"/>
+            <field name="state">code</field>
+            <field name="code">model.cron_compute_state()</field>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
```

Comment:
I think you can compute hourly, it's fairly light.

Reply:
Ok